.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: required
+dist: trusty
+
+os: linux
+
+language: c
+
+compiler:
+  - gcc
+  - clang
+
+env:
+  global:
+    - LUAJIT_LIB=/usr/lib64/libluajit-5.1.so
+    - LUAJIT_INC=/usr/include/luajit-2.0
+  matrix:
+    - NGINX_VERSION=1.9.14
+    - NGINX_VERSION=1.8.1
+    
+before_install:
+  - sudo apt-get install -qq -y cpanminus libluajit-5.1-dev
+  - sudo cpanm -v --notest Test::Nginx
+  - sudo cpanm -v --notest Test::Nginx::Socket
+  
+install:
+  - wget http://nginx.org/download/nginx-${NGINX_VERSION}.tar.gz && tar -xzf nginx-${NGINX_VERSION}.tar.gz
+  - git clone https://github.com/openresty/echo-nginx-module.git
+  - git clone https://github.com/openresty/lua-nginx-module.git
+  - git clone https://github.com/openresty/nginx-eval-module.git
+    
+script:
+  - cd nginx-${NGINX_VERSION}/
+  -  ./configure --with-debug --with-pcre --with-pcre-jit --with-http_dav_module --add-module=.. --add-module=../echo-nginx-module --add-module=../lua-nginx-module --add-module=../nginx-eval-module
```

Comment:
I think the module order is wrong. Please see the command in `util/build.sh` for the command I use for development :)

Also, will you break this long line into multiple lines? Possibly with `\`? It would make diff's much easier to read :)